### ISSUE TITLE
LineItems should be added to more payment methods than Klarna

### ISSUE THREAD

**Issue by Fifciu** [commercetools/commercetools-adyen-integration#728]
Hello,
I see in the code the extension module is adding `lineItems` out of the box for the Klarna payments which is great - However, [Adyen's doc](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/payments__reqParam_lineItems) says it is also required for a bunch of different payment methods:
![image](https://user-images.githubusercontent.com/30155292/121154676-7076d000-c847-11eb-8e0d-b2b94026fd3a.png)

I think it would be really great to add `lineItems` in case of these too.

**Comment by butenkor** [comment]
I agree that it would be helpful to add those infos in the integration automatically (similar as we did for Klarna) but I also would like to mention that `lineItems` and other required data can be provided by the users of the integration within the `makePaymentRequest` too. 
As a side note especially for the US customers it might not be trivial to generate accurate line item tax values since those often are calculated through the custom business logic or 3rd party services.

**Comment by Fifciu** [comment]
That's true. However, I would still go for generating it in `commercetools-adyen-integration` always if possible. Developer will provide own `lineItems` if needs it.

**Comment by Fifciu** [comment]
What do you think @butenkor ?

**Comment by lojzatran** [comment]
Hey @Fifciu starting from [version 9.1.0](https://github.com/commercetools/commercetools-adyen-integration/releases/tag/v9.1.0) we also add `lineItems` to Affirm.

**Comment by Fifciu** [comment]
Hi @lojzatran. I've started to generate lineItems on my side (using your logic for that) for every payment method. It is an optional field so it doesn't make me problems and based on this information I am getting better data in Adyen like Shopper's DNA or AVS Validation

**Comment by ahmetoz** [comment]
The feature described here is now supported with the release [v9.7.0](https://github.com/commercetools/commercetools-adyen-integration/releases/tag/v9.7.0)
